spec/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
```

Comment:
- [ ] Also, As stated in the requirements, we need to add unit tests for the `User`, `Post`, `Comment`, and `Like` models, currently, I could not find a test/spec file for the  `Comment`, and `Like` models. Please add this file and make sure to test for `validation` alongside other custom methods inside those models.

Reply:
Fixed.